erpnext/accounts/report/general_ledger/general_ledger.py

```diff
@@ -128,6 +128,10 @@ def get_gl_entries(filters):
 		filters['company_fb'] = frappe.db.get_value("Company",
 			filters.get("company"), 'default_finance_book')
 
+	dim_fields = ""
+	if filters.get("include_dims"):
+		dim_fields = f", {', '.join(get_accounting_dimensions())}"
```

Comment:
```suggestion
		dimension_fields = ', '.join([dimension_fields] + get_accounting_dimensions(as_list=1))
```

Reply:
the dimension_fields string must have the same number of commas in it as it has fields, otherwise there is no way to interpolate it into the query without possibly introducing a spurious empty field. I can see how "voucher_no{dim_fields}" did not look nice in the format below, so I am moving the additional comma to the end of the dimension_fields string. Hopefully that will make the format look more readable.

Reply:
Also, as_list=1 is the default for get_accounting_dimensions, so it would seem slightly odd to specify it. (Since the usual use is occurring, why put in a notation that might suggest to the code reader that the parameter is being supplied for a particular reason?)